modules/fedimint-wallet-tests/tests/tests.rs

```diff
@@ -192,6 +192,10 @@ async fn on_chain_peg_in_and_peg_out_happy_case() -> anyhow::Result<()> {
         deposit_updates.next().await.unwrap(),
         DepositStateV2::WaitingForConfirmation { .. }
     ));
+    assert!(matches!(
+        deposit_updates.next().await.unwrap(),
+        DepositStateV2::Confirmed { .. }
+    ));
```

Comment:
nit: would be nice if we can test that after `Claimed` state is emitted, the balance is correct (which would verify that all the e-cash has been issued)

Reply:
I added a commit to that end